README.md

```
@@ -2,36 +2,356 @@
-the OpenMS API to enable rapid algorithm development and workflow development.
-pyOpenMS supports the Proteomics Standard Initiative (PSI) formats for MS data. 
-
+LC-MS data management and analyses. 
 This repository contains documentation, installation instructions, and example code
 that show different functions of pyOpenMS.
 
-Installation
-=============
+## Table of Contents
+
+- [Installation](#installation)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)